modules/imap/js_modules/utils/handleNexterDateAction.js

```diff
@@ -0,0 +1,55 @@
+function handleNexterDateAction(e) {
+    let reload_and_redirect = function () {
+        Hm_Folders.reload_folders(true);
+        let path = hm_list_parent() ? hm_list_parent() : hm_list_path();
+        window.location.replace('?page=message_list&list_path=' + path);
```

Comment:
We don't do this to change routings. You should use [navigate()](https://github.com/cypht-org/cypht/blob/master/modules/core/navigation/navigation.js#L47)